### PR TITLE
feat(ramses): add app

### PR DESCRIPTION
**Pull request by tonzgao**
## Description

https://www.ramses.exchange/home
https://defillama.com/protocol/ramses-exchange

note: not affiliated with project

## Checklist

- [X] I have followed the [Contributing Guidelines](../CONTRIBUTING.md)
- [X] (optional) As a contributor, my Ethereum address/ENS is: tonygao.eth

## How to test?

example random address: 0x47c63ec6ff7ea416c56be4478eee93d32068b708

### CODE REVIEW

File: src/apps/ramses/common/ramses.voting-rewards.contract-position-fetcher.ts [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+      const tokenFound = baseTokens.find(p => p.address === token.toLowerCase());
+      if (!tokenFound) return null;
+      return {
+        metaType: MetaType.CLAIMABLE,
+        address: token,
+        network: this.network,
+      };
+    });
+
+    return _.compact(tokenDefinitions);
```
[the remark this answers is not in the document]
Comment (tonzgao) [reply src/apps/ramses/common/ramses.voting-rewards.contract-position-fetcher.ts:43]: this is a copy pasta from velodrome and other solidly forks, so idk about the rationale behind a lot of things. but in general, rewards are permissionless so can be anything.

[the remark this answers is not in the document]
Comment (wpoulin) [reply src/apps/ramses/common/ramses.voting-rewards.contract-position-fetcher.ts:43]: sg no worries, I'll merge it right away. I thought you used it cause some reward tokens were missing

**Comment by SamIam-0x** [comment]
Hey @tonzgao -- thanks for the integration! FYI I do not see a Ramses value in the random address provided at https://zapper.xyz/account/0x47c63ec6ff7ea416c56be4478eee93d32068b708?tab=apps

Though, I do see them propagating for this other random address -https://zapper.xyz/account/0x865d7eb5db37cc02ec209dd778f4e3851a421a20?tab=apps

Is that consistent with expected behavior?

**Comment by tonzgao** [comment]
> Hey @tonzgao -- thanks for the integration! FYI I do not see a Ramses value in the random address provided at https://zapper.xyz/account/0x47c63ec6ff7ea416c56be4478eee93d32068b708?tab=apps
> 
> Though, I do see them propagating for this other random address -https://zapper.xyz/account/0x865d7eb5db37cc02ec209dd778f4e3851a421a20?tab=apps
> 
> Is that consistent with expected behavior?

yeah, mercenary capital. 

one thing though is that the name is currently "ramses" and should be capitalized. idk if this is worth creating a pr for though.

**Comment by SamIam-0x** [comment]
No worries, just changed that!

**Pull request merged**